src/Coin.cpp

```diff
@@ -289,6 +297,15 @@ DerivationPath TW::derivationPath(TWCoinType coin) {
             DerivationPathIndex(0, true),
         };
 
+    case TWCoinTypeCosmos:
```

Comment:
This is same as previous `DerivationPath(purpose(coin), coin, 0, 0, 0);`
Can you cross checking with ledger to make sure the derivation path is same?